torch/utils/throughput_benchmark.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch._C
+
+class ThroughputBenchmark(object):
+    '''
+    This class is a wrapper around a c++ component throughput_benchmark::ThroughputBenchmark
+    responsible for executing a PyTorch module (nn.Module or ScriptModule)
+    under an inference server like load. It can emulate multiple calling threads
+    to a single module provided. In the future we plan to enhance this component
+    to support inter and intra-op parallelism as well as multiple models
+    running in a single process.
```

Comment:
1. No one cares if this is a C++ binding or not. Describe what this thing does in the docs, not the implementation details.
2. Future plans do not go into the docs. When you add the extra features (like intra-op parallelism), you'll update the docs. Leave this out for now.
3. Why is this specific to a server-like deployment? Measuring throughput of models doesn't require that, and it's a valid thing to do while performing research.

Reply:
1. I disagree . This is a part of showing how we designed things and why. It is the next level comparing to docs which don't answer the "why" question
2. They might. We show future developers how to modify this code.
3. I think there is a misunderstanding here , ideally we discuss this in person if you are interested. Tl/Dr - one model many callers is pretty specific to deployment scenario where you have an inference service 
We might be interested in this on other scenarios, I agree. That's why I use "like" here